### PR TITLE
Bump AVM to research-8.1.0

### PR DESCRIPTION
**Pull request by vrabaud**

### CODE REVIEW

**Review by maryla-uc — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The changes are relative to the previous release, unless the baseline is specifi
   decoder->image->gainMap != NULL instead.
 * Write an empty HandlerBox name field instead of "libavif" (saves 7 bytes).
 * Update aom.cmd/LocalAom.cmake: v3.11.0
-* Update avm.cmd: research-v8.0.0
+* Update avm.cmd: research-v8.1.0
```
Comment (wantehchang) [inline CHANGELOG.md:31]: @y-guyon 

In src/codec_avm.c, we have a TODO comment:

```
        // avm does not handle monochrome as of research-v8.0.0.
        // TODO(yguyon): Enable when fixed upstream
        codec->internal->monochromeEnabled = AVIF_FALSE;
```

Do you know if this bug has been fixed in research-8.1.0? Did we file an issue on this bug?

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks!

File: src/obu.c [hunk 2]
```diff
@@ -388,41 +387,6 @@ static avifBool parseAV2SequenceHeader(avifBits * bits, avifSequenceHeader * hea
...
-    //   separate_uv_delta_q
-    //   base_y_dc_delta_q
-    //   base_uv_dc_delta_q
-
-    // See read_sequence_header_obu() in avm.
-    // Ignored field.
-    //   film_grain_params_present
-
-    // See av1_read_sequence_header_beyond_av1() in avm.
-    // Other ignored fields.
```
Comment (y-guyon) [inline src/obu.c:423]: Some of these comments should be kept, see https://gitlab.com/AOMediaCodec/avm/-/blob/main/av1/decoder/obu.c#L262

**Review by y-guyon — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (y-guyon) [reply CHANGELOG.md:31]: > Do you know if this bug has been fixed in research-8.1.0?

It is not fixed in `research-8.1.0`.  
@vrabaud Maybe update the comment in `src/codec_avm.c` to 8.1.0?

> Did we file an issue on this bug?

https://gitlab.com/AOMediaCodec/avm/-/issues/522

Comment (vrabaud) [reply CHANGELOG.md:31]: Done

File: src/obu.c (hunk 2 above)
Comment (vrabaud) [reply src/obu.c]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (y-guyon) [reply CHANGELOG.md:31]: Thank you.

Tentative fix: https://gitlab.com/AOMediaCodec/avm/-/merge_requests/1396

**Pull request merged**